### PR TITLE
Modify css

### PR DESCRIPTION
**Pull request by baymac**
## Problem

_What problem are you trying to solve?_

The previous website needed some UI changes and code refractory. For displaying the code images were used, now code UI has been implemented.
## Solution

_How did you solve the problem?_

I have modified the existing design with different colors, separate stylesheets for the article segment, added link for social media handles, used js for slider function, removed images and replaced it with code blocks.

## Before & After Screenshots

**BEFORE**:
screenshot from 2019-02-02 18-31-42

**AFTER**:
screenshot from 2019-02-02 18-31-17

### CODE REVIEW

**Comment by baymac** [comment]
You can check the website hosted on my heroku account. Link: https://proxy-master-baymac.herokuapp.com/

**Comment by baymac** [comment]
After changing colors and instructions format. It looks like this.
screenshot from 2019-02-03 06-18-55

File: templates/index.html [hunk 1]
```diff
@@ -1,48 +1,79 @@
...
-        	<p style="font-size: 120%"><b>To Unset Proxy:</b></p>
-        	<img src="../static/UnsettingProxy.png">
-        </fieldset>
+
+    <h3>Known Issues</h3>
+  	<p>
+  		Some Users might need to change <strong>Settings->Network->Proxy</strong> from <strong>Off</strong> to <strong>Manual</strong>.
+  	</p>
+    <div class="share">
+      <a href=""><img src="../static/img/facebook.svg" alt=""></a>
```
Comment (thealphadollar) [inline templates/index.html]: https://www.facebook.com/metakgp/

File: templates/index.html [hunk 2]
```diff
@@ -1,48 +1,79 @@
...
+      <a href=""><img src="../static/img/twitter.svg" alt=""></a>
```
Comment (thealphadollar) [inline templates/index.html]: https://twitter.com/metakgp

File: templates/index.html [hunk 3]
```diff
@@ -1,48 +1,79 @@
...
+      <a href=""><img src="../static/img/github.svg" alt=""></a>
```
Comment (thealphadollar) [inline templates/index.html]: https://github.com/metakgp

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Please add the following links in the social media buttons.

File: templates/index.html (hunk 3 above)
Comment (baymac) [reply templates/index.html]: I have added them.

**Review by thealphadollar — APPROVED** [review]
LGTM!

**Comment by thealphadollar** [comment]
Thank you @baymac for the PR.

**Pull request merged**